CONTRIBUTING.md

```diff
@@ -85,6 +85,31 @@ function startFoo() {
 }
 ```
 
+### Checking for `undefined` or `null`
+
+In node-wot, we enabled [strict boolean expressions](https://typescript-eslint.io/rules/strict-boolean-expressions/). In summary, this means that in the
+the code base is not allowed to use non-boolean expressions where a boolean is expected (see the [examples](https://typescript-eslint.io/rules/strict-boolean-expressions/#examples)).
+How then should the contributor deal with nullable variables? For example:
+
+```ts
+function(arg1: string | null | undefined) {
+    // ERROR: not allowed by strict-boolean-expressions
+    if(!arg) { throw new Error("arg should be defined!))}
+}
+```
+
+Instead of checking for both null and `undefiend` values (`if(arg !== undefined && arg !== null)`) the preferred solution is to use `!=` or `==` operator. Interestingly in JavaScript
+with ==, null and undefined are only equal to each other. Example:
```

Comment:
```suggestion
Instead of checking for both null and `undefined` values (`if (arg !== undefined && arg !== null)`) the preferred solution is to use `!=` or `==` operator. Interestingly in JavaScript
with `==`, `null` and `undefined` are only equal to each other. Example:
```